README.md

```diff
@@ -50,33 +61,7 @@ All enums should live in their own code file, except in cases where the enum is
 
 #### Usage of MARK ####
 
-To help organizing your files you may want to use pragma marks to clearly separate your functions, properties and extensions. Only use them for the groups defined above. For extensions, use one MARK per extension. For example, `// MARK: UITableViewDelegate` instead of `// MARK: Extensions`
-
-```swift
-// MARK: Functions
-
-override func viewDidLoad() {
-}
-
-public func doSomething() {
-}
-
-internal func doSomething() {
-}
-
-private func doSomethingElse() {
-}
-
-// MARK: UITableViewDelegate Extension
-
-extension UIViewController: UITableViewDelegate {
-}
-
-// MARK: UITableViewDataSource Extension
-
-extension UIViewController: UITableViewDataSource {
-}
-```
+To help organizing your files you may want to use pragma marks to clearly separate your functions, properties and extensions. For extensions, use one MARK per extension. For example, `// MARK: UITableViewDelegate` instead of `// MARK: Extensions`.
```

Comment:
I think the example before was too confusing that's why I removed it. Also since we don't really know if and how people will use it I don't want to restrain anything until we have a better idea on how to use them. I still think it's interesting to have it in the style guide to suggest ways of organizing your files, it's about file structure so not mentioning MARKs would be weird to me